Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```diff
@@ -0,0 +1,228 @@
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionBody",
+                "replaceTokensInText_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "input" : {
+                "userCommand" : "createBluePrint",
+                "assignTo" : "$(BLUEPRINT_FILENAME)"
+            },
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintFile",
+                "replaceTokensInText_BlueprintFile",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "writeFile_BlueprintFile",
+                "appendFileContent_BlueprintBody",
+                "readFileContent_BlueprintImport",
+                "replaceTokensInText_BlueprintImport",
+                "writeFile_BlueprintImport",
+                "appendFileContent_FunctionApp",
+                "readFileContent_BlueprintRegister",
+                "replaceTokensInText_BlueprintRegister"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the Blueprint",
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "appendFileContent_BlueprintBody"
+            ]
+        }
+    ],
+    "actions" : [
+        {
+            "name": "getFunctionName",
+            "type": "UserInput",
+            "assignTo": "$(FUNCTION_NAME_INPUT)",
+            "paramId": "trigger-functionName",
+            "defaultValue": "EventhubTrigger"
+        },
+        {
+            "name": "getEventHubName",
+            "type": "UserInput",
+            "assignTo": "$(EVENTHUB_NAME)",
+            "paramId": "trigger-eventhubName",
+            "defaultValue": "MyEventHub"
+        },
+        {
+            "name": "getConnnectionString",
+            "type": "UserInput",
+            "assignTo": "$(EVENTHUB_CONNECTION_STRING)",
+            "paramId": "trigger-connectionStringName",
+            "defaultValue": ""
+        },
+        {
+            "name": "readFileContent_FunctionApp",
+            "type": "ReadFromFile",
+            "assignTo": "$(FUNCTION_APP_CONTENT)",
+            "filePath" : "function_app.py"
+        },
+        {
+            "name": "replaceTokensInText_FunctionApp",
+            "type": "ReplaceTokensInText",
+            "source": "$(FUNCTION_APP_CONTENT)",
+            "assignTo" : "$(FUNCTION_APP_CONTENT)"
+        },
+        {
+            "name": "writeFile_FunctionApp",

```

Comment:
A lot of these actions don't seem unique to event hub and/or python. Can we add an "actions.json" file as a place to share actions? I'm not saying we get rid of this "actions" array, though - because I think it could still be useful to have template-specific actions stored here.

Reply:
We considered adding actions.json as a place to share action, however at this point we decided to keep the actions as is.